### ISSUE TITLE
Lenovo Yoga 710, control enabled, target 0%, still blowing +-70%

### ISSUE THREAD

**Issue by mgkai** [hirschmann/nbfc#766]
TNX hirschmann for including a setting for Yoga710
It keeps blowing away though; often on and off, on and off, like a sinus wave.
Not very helpful during work
Out of desperation I moved fan control slider all the way to to 0%, also updated to latest version 1.63; result cf above.
Is there no Save button or anything to activate the setting?
Sure, fan control status is "enabled"

**Comment by keeferrourke** [comment]
I have this same problem!

**Comment by mgkai** [comment]
Two is a company... I noticed that I had 100% processor performance specified in energy saving plan that is active on my yo710, and issue is much better / fan activity appears reasonable after I tuned that down to 90%

**Comment by mgkai** [comment]
Besides, I tried to do everything with the Geforce graphic processor and disabled the CPU-internal Intel wherever possible. As I felt that the yo710 was less performant when both graphic processors were active

**Comment by keeferrourke** [comment]
I have the issue running on Linux where `nfbc set -s 0` doesn't always get respected. The fan will come on in bursts at times. The laptop doesn't actually get hot, but the controller seems really sensitive or is ignoring NBFC. Is there a way to *disable* the fan?

**Comment by larshb** [comment]
Are you guys able to read the fan speed at all? I posted a separate [issue](https://github.com/hirschmann/nbfc/issues/808) / bug report for my particular problems.

I also turned down processor performance settings to 90 % like @mgkai suggests. Can you give any more detail on what "disabling CPU-internal Intel" entails?

**Comment by mgkai** [comment]
Confirm; I am able to see fan speed percentage, in system tray and in main app window. --
The 710 comes with Intel integrated and Nvidia Geforce GPU. Standard windows setting is to use either of them "automatically". I felt there was even more fan-blowing whenever (apparently most of the time) Windows decided to use Intel GPU, compared to after I instructed Windows to use the Nvidia as a standard. To do so, you can use Nvidia control panel, or right-click on any *EXE file, eg on NoteBookFanControl.exe, then select the context menu entry "Execute with Graphic processor" and choosing Nvidia from its submenu.

**Comment by larshb** [comment]
That's strange. I do not have that context menu entry and whenever I try to launch NVIDIA Control Panel it just tells me I do not have an NVIDIA-card:

![image](https://user-images.githubusercontent.com/8710787/71962352-8834c980-31f9-11ea-8752-af3d1a217391.png)

The error-dialog above is "Can't find an NVIDIA graphic card in the system" in Norwegian. I may be missing a driver, but this feels like a red herring to me.

My graphics card is recognized (by `dxdiag`) as Intel(R) HD Graphics 620 and I am using Intels driver 26.20.100.7584 (from november 2019). Is there some NVIDIA-driver that would be compatible as an alternative?

**Comment by mgkai** [comment]
I can't exclude that Lenovo may have sold some versions of the 710 without dedicated graphics in some markets. Which would be unfortunate as a dedictated graphics card makes all the difference in my experience. So why not give it a try and check to see if just the driver is missing. Lenovo also offers [driver download/update services](https://pcsupport.lenovo.com/de/de/products/laptops-and-netbooks/yoga-series/yoga-710-14ikb/80v4/80v4006gge/mp17avag/downloads/automatic-driver-update).

**Comment by mgkai** [comment]
So I am being told by mail that this thread will be closed because nobody commented on it any more. Well then, I did not open the topic just for chatting fun but looking for developer input, hoping for a solution. Since this has not materialized, for me and for others here, I will do the closing remarks myself; in summary, forget all the twaeking and head-scratching, rather get yourself a better machine, as I eventually did. I gave up on the 710 and Lenovo in general, coming to believe that their "drivers work" is not "clean" enough. My new laptop is whisper-quiet under similar load as before, even now it is summertime. Goodbye Lenovo!
